stage8/00-license-generation/00-run.sh

```diff
@@ -261,14 +261,63 @@ echo "<th>Location</th>" >> ${FILE}
 echo "</tr>" >> ${FILE}
 echo "</thead>" >> ${FILE}
 echo "<tbody>" >> ${FILE}
-package_table_items $((var++)) "Linux" "6.1" "GPLv2" "https://github.com/analogdevicesinc/linux"
-package_table_items $((var++)) "HDL"   " "    "GPL/LGPL/BSD" "https://github.com/analogdevicesinc/hdl"
-package_table_items $((var++)) "LibIIO" " " "LGPL-2.1/GPL-2.0" "https://github.com/analogdevicesinc/libiio"
-package_table_items $((var++)) "IIO Oscilloscope" " " "GPL-2.0" "https://github.com/analogdevicesinc/iio-oscilloscope"
-package_table_items $((var++)) "Scopy" " " "GPL-3.0" "https://github.com/analogdevicesinc/scopy"
+
+package_table_items $((var++)) "linux" "6.1" "GPLv2" "https://github.com/analogdevicesinc/linux"
+wget https://raw.githubusercontent.com/analogdevicesinc/linux/main/COPYING
+html_pre_file COPYING linux
+rm COPYING
+
+package_table_items $((var++)) "hdl"   " "    "GPL/LGPL/BSD" "https://github.com/analogdevicesinc/hdl"
+wget https://raw.githubusercontent.com/analogdevicesinc/hdl/main/LICENSE
+html_pre_file LICENSE hdl
+rm LICENSE
+
+package_table_items $((var++)) "libiio" " " "LGPL-2.1/GPL-2.0" "https://github.com/analogdevicesinc/libiio"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/libiio/COPYING.txt libiio
+
+package_table_items $((var++)) "iio-oscilloscope" " " "GPL-2.0" "https://github.com/analogdevicesinc/iio-oscilloscope"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/iio-oscilloscope/LICENSE iio-oscilloscope
+
+package_table_items $((var++)) "scopy" " " "GPL-3.0" "https://github.com/analogdevicesinc/scopy"
+wget https://raw.githubusercontent.com/analogdevicesinc/scopy/main/LICENSE
+html_pre_file LICENSE scopy
+rm LICENSE
+
 package_table_items $((var++)) "libad9361-iio" " " "GPL-2.0" "https://github.com/analogdevicesinc/libad9361-iio"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/libad9361-iio/LICENSE libad9361-iio
+
 package_table_items $((var++)) "libad9166-iio" " " "GPL-2.0" "https://github.com/analogdevicesinc/libad9166-iio"
-package_table_items $((var++)) "fru-tools" " " "GPL-2.0" "https://github.com/analogdevicesinc/fru-tools"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/libad9166-iio/LICENSE libad9166-iio
+
+package_table_items $((var++)) "fru_tools" " " "GPL-2.0" "https://github.com/analogdevicesinc/fru_tools"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/fru_tools/license.txt fru_tools
+
+package_table_items $((var++)) "pyadi-iio" " " "ADI BSD" "https://github.com/analogdevicesinc/pyadi-iio"
```

Comment:
The version of pyadi-iio should be extracted automatically by package manager - even if not, we are using a release tag, not building latest main - so can hardcode a version here.